src/test/java/edu/cornell/kfs/vnd/document/fixture/WorkdayKfsVendorLookupRootEnum.java

```diff
@@ -0,0 +1,66 @@
+package edu.cornell.kfs.vnd.document.fixture;
+
+import java.util.Calendar;
+
+import org.kuali.kfs.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.service.impl.TestDateTimeServiceImpl;
+import edu.cornell.kfs.vnd.document.CuVendorMaintainableImplTest;
+import edu.cornell.kfs.vnd.jsonobject.WorkdayKfsVendorLookupResult;
+import edu.cornell.kfs.vnd.jsonobject.WorkdayKfsVendorLookupRoot;
+
+public enum WorkdayKfsVendorLookupRootEnum {
+    EMPTY_LIST(),
+    ACTIVE("active1", "1", true, true, false),
+    INACTIVE_NO_TERM_DATE("inactive1", "0", false, true, true),
+    RECENT_TERMINATED("inactive2", "0", CuVendorMaintainableImplTest.TERMINATION_DAYS_IN_THE_PAST - 3, false, true, true),
+    TERMINATED_AT_MAX("inactive3", "0", CuVendorMaintainableImplTest.TERMINATION_DAYS_IN_THE_PAST, false, true, true),
+    OLD_TERMINATED("inactive4", "0", CuVendorMaintainableImplTest.TERMINATION_DAYS_IN_THE_PAST + 5, false, false, false);
+    
+    public final WorkdayKfsVendorLookupRoot lookupRoot;
```

Comment:
The immutable fixture enum contains a field that holds a mutable object. Is that going to cause potential issues? Does the enum need a helper instance method for generating a `WorkdayKfsVendorLookupRoot` instance upon request?

Reply:
ok